Source/Extensions/Blazorise.DataGrid/_DataGridCellDateEdit.razor

```diff
@@ -0,0 +1,41 @@
+@using Blazorise.Utilities
+@typeparam TItem
+@inherits ComponentBase
+
+@if ( TypeHelper.IsDate( valueType ) )
+{
+    @dateFragment
+}
+
+@code {
+    private RenderFragment dateFragment => builder =>
+    {
+        var type = typeof( DateEdit<> ).MakeGenericType( new[] { valueType } );
+
+        builder.OpenComponent( 0, type );
+        builder.AddAttribute( 1, nameof( DateEdit<object>.Date ), valueType switch
+        {
+            Type typeDateTime when typeDateTime == typeof( DateTime ) => (DateTime)( CellValue ?? (DateTime)default ),
```

Comment:
I haven't checked. But how performant is all this checking for a type? Can we do the check during OnInitialized, store the information in local varaobles, and then just render based on that?

Reply:
I don't suppose it will be a problem, though, since this is only rendered in a table header, so not a lot of render cycles should be made.